### PR TITLE
feat: add statement metrics

### PR DESCRIPTION
**Pull request by nchamo**
We are making a few interesting changes here:
1. We moved all custom postgres config into a `postgresql.conf` file
2. We added some new custom metrics to postgres_exporter. 
    * We added stats about statements, such as min, avg and max time spent in a statement
    * We also added stats about tables, such as number of rows added/updated/deleted
3. We added a new way to execute migrations to our postgres, and we used it to install the extension used for statements stats

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -291,7 +291,7 @@ fi
 
 docker pull "decentraland/katalyst:${DOCKER_TAG:latest}"
 if [ $? -ne 0 ]; then
-  echo -n "Failed to stop nginx! "
+  echo -n "Failed to pull the docker image with tag ${DOCKER_TAG:latest}"
```
Comment (nchamo) [inline init.sh:294]: We were printing the wrong error here

File: local/postgres/postgresql.conf [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+
+#------------------------------------------------------------------------------
+# RESOURCE USAGE (except WAL)
+#------------------------------------------------------------------------------
+
+# - Memory -
+
+#shared_buffers = 32MB			# min 128kB
+					# (change requires restart)
+shared_buffers = 2560MB
```
Comment (nchamo) [inline local/postgres/postgresql.conf:123]: I left the defaults/explanations commented, and just added the values below. Makes sense?

File: docker-compose.yml [hunk 3]
```diff
@@ -34,34 +34,25 @@ services:
...
-      -c default_statistics_target=100
-      -c random_page_cost=4
-      -c effective_io_concurrency=2
-      -c work_mem=64MB
-      -c min_wal_size=1GB
-      -c max_wal_size=4GB
-      -c max_worker_processes=3
-      -c max_parallel_workers_per_gather=2
-      -c max_parallel_workers=3
-      -c max_parallel_maintenance_workers=2
```
Comment (nchamo) [inline docker-compose.yml:53]: All these were moved to the `.conf` file

File: local/postgres-exporter/queries.yaml [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    - datname:
+        usage: "LABEL"
+        description: "Name of current database"
+    - schemaname:
+        usage: "LABEL"
+        description: "Name of the schema that this table is in"
+    - relname:
+        usage: "LABEL"
+        description: "Name of this table"
+    - seq_scan:
```
Comment (menduz) [inline local/postgres-exporter/queries.yaml:38]: will those metrics have a prefix? or we will query `seq_scan` in prometheus?

File: local/postgres-exporter/queries.yaml [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    - total_time_seconds:
+        usage: "COUNTER"
+        description: "Total time spent in the statement, in milliseconds"
+    - min_time_seconds:
+        usage: "GAUGE"
+        description: "Minimum time spent in the statement, in milliseconds"
+    - max_time_seconds:
+        usage: "GAUGE"
+        description: "Maximum time spent in the statement, in milliseconds"
+    - mean_time_seconds:
```
Comment (menduz) [inline local/postgres-exporter/queries.yaml:121]: Same here, if metrics don't have a prefix, the name is very ambiguous `pg_query_mean_time_seconds` sounds better in a dashboard than `mean_time_seconds`

**Review by menduz — CHANGES_REQUESTED** [review]

**Review by menduz — APPROVED** [review]

**Review by agusaldasoro — APPROVED** [review]

**Pull request merged**